inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/scheduling/tasks/PredictionTask.java

```diff
@@ -62,66 +73,88 @@ public void run()
     {
         User user = getUser();
 
-        Predictions model = new Predictions(getProject(), getUser()); 
+        Project project = getProject();
+        Predictions model = new Predictions(project, getUser());
+        List<AnnotationDocument> documents = documentService.listAnnotationDocuments(project, user);
```

Comment:
@Rentier `listAnnotationDocuments` is not adequate because it only returns annotation documents that were opened by the user at least once. Hence we are not guaranteed to get predictions for all documents here. It is necessary to iterate over the source documents and then obtain the annotation CAS for each source document for the given user. See https://github.com/webanno/webanno/pull/1025

The same probably goes for the evaluation task.